src/win32/ossec.conf

```diff
@@ -128,9 +128,10 @@
     <directories check_all="yes">%WINDIR%\System32\wbem\WMIC.exe</directories>
     <directories check_all="yes">%WINDIR%\System32\WindowsPowerShell\v1.0\powershell.exe</directories>
     <directories check_all="yes">%WINDIR%\System32\winrm.vbs</directories>
-
     <directories check_all="yes" realtime="yes">%PROGRAMDATA%\Microsoft\Windows\Start Menu\Programs\Startup</directories>
 
+    <ignore>c:\programdata\microsoft\windows\start menu\programs\startup\desktop.ini</ignore>
```

Comment:
Can't we use the relative path as the same way as the directory set above? `%PROGRAMDATA%\Microsoft\Windows\Start Menu\Programs\Startup\desktop.ini`